flow360/component/flow360_params/flow360_output.py

```diff
@@ -436,7 +451,7 @@ class IsoSurfaceOutput(Flow360BaseModel, AnimatedOutput):
     iso_surfaces: Optional[IsoSurfaces] = pd.Field(alias="isoSurfaces")
 
 
-class AeroacousticOutput(Flow360BaseModel):
+class AeroacousticOutput(Flow360BaseModel, AnimatedOutput):
```

Comment:
I think aeroacoustic does not support animations

Reply:
![image](https://github.com/flexcompute/Flow360/assets/146361853/19dedbaf-1337-4b0e-bf06-2e06e528d00f)
Funnily enough the schema mentions them while the documentation does not - will remove.

Reply:
Left it in as it seems to be present in example files (and schema)